examples/app/crossfilter/models/styleable_box.coffee

```diff
@@ -39,15 +39,10 @@ class StyleableBox extends BaseBox.Model
   type: "StyleableBox"
   default_view: StyleableBoxView
 
-  defaults: ->
-    return _.extend {}, super(), {
-      children: []
-      css_properties: {}
-      orientation: 'vertical'
-    }
-
-  children: () ->
-    return @get('children')
+  @define {
+    css_properties: [p.Dict, null]
+    orientation:    [p.String, 'vertical']
```

Comment:
Ah nice, yes I'll switch an use that...

On Mon, Apr 4, 2016 at 3:49 PM, Mateusz Paprocki <notifications@github.com>
wrote:

> In examples/app/crossfilter/models/styleable_box.coffee
> <https://github.com/bokeh/bokeh/pull/4105#discussion_r58446258>:
>
> > @@ -39,15 +39,10 @@ class StyleableBox extends BaseBox.Model
> >    type: "StyleableBox"
> >    default_view: StyleableBoxView
> >
> > -  defaults: ->
> > -    return _.extend {}, super(), {
> > -      children: []
> > -      css_properties: {}
> > -      orientation: 'vertical'
> > -    }
> > -
> > -  children: () ->
> > -    return @get('children')
> > +  @define {
> > +    css_properties: [p.Dict, null]
> > +    orientation:    [p.String, 'vertical']
>
> You could use p.Orientation enum property. Unless this orientation has
> more choices.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/bokeh/bokeh/pull/4105/files/6063bc5260cc3e98f14eb505b63e784bf599c043#r58446258>
>